tools/SeeDot/faceDetection.md

```diff
@@ -0,0 +1,114 @@
+# Quantizing Face Detection using SeeDot
```

Comment:
Replace "Quantizing" with "Quantized" here please.

Reply:
Since the document is discussing the steps for quantizing won't 'Quantizing' sound better?

Reply:
Nope. Quantization is a process. You quantize neural networks (an object), how would you quantize face detection (a problem/concept)?